recipes-devtools/python-jsonref/python-jsonref.inc

```diff
@@ -6,6 +6,6 @@ LIC_FILES_CHKSUM = "file://LICENSE;md5=a34264f25338d41744dca1abfe4eb18f"
 SRC_URI[md5sum] = "42b518b9ccd6852d1d709749bc96cb70"
 SRC_URI[sha256sum] = "f3c45b121cf6257eafabdc3a8008763aed1cd7da06dbabc59a9e4d2a5e4e6697"
 
-inherit pypi setuptools
```

Comment:
a) delete python-jsonref.inc or b) don't remove setuptools as it will introduce regression

Reply:
Hi @artur-gebicz-ml, python-jsonref.inc is included in recipe-devtools/python-jsonref/python3-jsonref_0.2.bb
also we cannot add setuptools, since this header file is used inside python3-jsonref_0.2.bb, which is inheriting  setuptools3

Reply:
Oh, sorry my fault. Then tell me please what's the meaning of having separated *.bb file and *.inc? Wouldn't it be best to merge it together?

Reply:
Yea, no problem, since it is derived from python2 the the python-jsonref.inc kept as generic.